static/app/views/seerExplorer/sessionDropdown.tsx

```diff
@@ -0,0 +1,183 @@
+        </SessionMeta>
+      </SessionOption>
+    ),
+    value: session.run_id,
+    textValue: session.title, // Used for search.
+  };
+}
+
+interface SessionDropdownProps {
+  activeRunId: number | null;
+  onSelectSession: (runId: number | null) => void;
+  organization: Organization | null;
+  startNewSession: () => void;
+  useExplorerSessionsResult: ReturnType<typeof useExplorerSessions>;
+}
+
+export function SessionDropdown({
+  activeRunId,
+  useExplorerSessionsResult,
+  onSelectSession,
+  startNewSession,
+}: SessionDropdownProps) {
+  const {sessions, isFetching, isError, hasNextPage, fetchNextPage, isFetchingNextPage} =
+    useExplorerSessionsResult;
+
+  const selectOptions = sessions.map(makeSelectOption);
+
+  const makeTrigger = (
+    props: Omit<React.HTMLAttributes<HTMLElement>, 'children'>,
+    isOpen: boolean
+  ) => {
+    return (
+      <TriggerButton {...props} aria-label="Select session">
+        <IconChevron direction={isOpen ? 'up' : 'down'} size="xs" />
+      </TriggerButton>
+    );
+  };
+
+  const menuFooter = (
+    <Fragment>
+      {hasNextPage && (
+        <FooterWrapper>
+          <Button
+            size="xs"
+            onClick={() => fetchNextPage()}
+            disabled={isFetchingNextPage}
+            aria-label={t('Load more sessions')}
+          >
+            {isFetchingNextPage ? <LoadingIndicator size={12} /> : t('Load More')}
+          </Button>
+        </FooterWrapper>
+      )}
+    </Fragment>
+  );
+
+  const newSessionButton = activeRunId && (
+    <NewSessionButton size="xs" onClick={startNewSession}>
+      <Flex gap="2xs" align="center">
+        <IconAdd size="xs" />
+        {t('New')}
+      </Flex>
+    </NewSessionButton>
+  );
+
+  return (
+    <CompactSelect
+      key={activeRunId ?? 'new-session'} // Force re-render when runId changes.
+      searchable
+      menuWidth={350}
+      position="bottom-start"
+      value={activeRunId ?? undefined}
+      menuTitle={t('Session History')}
+      searchPlaceholder={t('Search sessions...')}
+      size="sm"
+      onChange={opt => {
+        onSelectSession(opt.value ?? null);
+      }}
+      options={selectOptions}
+      trigger={makeTrigger}
+      emptyMessage={
+        isError
+          ? t('Error loading sessions')
+          : isFetching
+            ? t('Loading sessions...')
+            : t('No sessions found.')
+      }
+      menuFooter={menuFooter}
+      menuHeaderTrailingItems={newSessionButton}
+    />
+  );
+}
+
+const TriggerButton = styled(Button)`
+  display: flex;
+  padding: ${space(0.75)} ${space(1.5)};
+  border: 1px solid ${p => p.theme.border};
+  border-radius: ${p => p.theme.borderRadius};
+  background: ${p => p.theme.background};
+  max-width: 300px;
+
+  &:hover {
+    background: ${p => p.theme.hover};
+  }
+`;
+
+const NewSessionButton = styled(Button)`
+  padding-top: 0;
+  padding-bottom: 0;
+  border: none;
+  background: transparent;
+`;
```

Comment:
I would strongly discourage you from doing this. You should use the button variants that are provided out of the box. At some point, we will remove the ability to style these too